ext/date/date.gemspec

```diff
@@ -1,7 +1,12 @@
 # frozen_string_literal: true
+
+version = File.foreach(File.expand_path("../lib/date.rb", __FILE__)).find do |line|
+  /^\s*VERSION\s*=\s*["'](.*)["']/ =~ line and break $1
```

Comment:
This regular expression accepts empty versions, such as ``VERSION = ''``.
Maybe it should only accept non-empty groups with ``(.+)`` or ``([^'"]+)``.
The last version could be simplified to ``/^\s*VERSION\s*=\s*["']([^'"]+)/``.

Reply:
Can you suggest it to https://github.com/ruby/date ? Thanks.